kubernetes/zenko/values.yaml

```diff
@@ -48,6 +33,25 @@ global:
   locationConstraints: {}
   replicationEndpoints: []
 
+maintenance:
+  # Enables a retry CronJobs that will attempt to retry CRR objects stuck in
+  # pending or failed state based off the specified schedule. For Orbit enabled
+  # installs, Zenko will need to be registered and account credentials
+  # (secretKey and accessKey) will need to be provisioned prior to enabling this
+  # maintenance feature.
+  enabled: false
+  accessKey:
```

Comment:
`""`?

Reply:
It doesn't require the empty quotes. Also it will fail silently but install if there are empty quotes and the maintenance is enabled. Where as it will fail to install if the values are empty and is attempted to be enabled.